AjobK/Athena
# 246-reCAPTCHA

closes #309 

<br />

 **What is it supposed to do**  
This branch adds reCAPTCHA to the login and register page

<br />

 **How can we test your branch**  

- Go to the login page 
- Try to log in without clicking on the checkbox
- You should get an error saying: "invalid reCAPTCHA"
- Now click on the checkbox
- You should be redirected to the user's profile page

<br />

 **Additional notes**  
I've decided to use a reCAPTCHA checkbox instead of the invisible reCAPTCHA because, the invisible reCAPTCHA functions badly and doesn't meet the minimum requirements.

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select atleast two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

> This form should be filled in by the author of this PR.
